apps/mobile/src/components/Community/CommunityMeta.tsx

```diff
@@ -70,39 +61,83 @@ export function CommunityMeta({ communityRef, queryRef }: Props) {
     graphql`
       fragment CommunityMetaQueryFragment on Query {
         ...isFeatureEnabledFragment
+        viewer {
+          ... on Viewer {
+            user {
+              __typename
+            }
+          }
+        }
       }
     `,
     queryRef
   );
 
+  const isMemberOfCommunityQuery = useLazyLoadQuery<CommunityMetaQuery>(
+    graphql`
+      query CommunityMetaQuery($communityID: DBID!) {
+        viewer {
+          ... on Viewer {
+            user {
+              isMemberOfCommunity(communityID: $communityID)
+            }
+          }
+        }
+      }
+    `,
+    {
+      communityID: community.dbid,
+    }
+  );
+
+  const isMemberOfCommunity = isMemberOfCommunityQuery.viewer?.user?.isMemberOfCommunity ?? false;
```

Comment:
Anyone know what's the best way to refetch this query? 🤔

Context: Would love to refetch this value whenever the sync is completed